script/optimize/optimize/entrypoint.sh

```diff
@@ -23,6 +23,7 @@ ORG_IMAGE_TAG="${REGISTRY_HOST}/test/test:org$(date '+%M%S')"
 OPT_IMAGE_TAG="${REGISTRY_HOST}/test/test:opt$(date '+%M%S')"
 NOOPT_IMAGE_TAG="${REGISTRY_HOST}/test/test:noopt$(date '+%M%S')"
 TOC_JSON_DIGEST_ANNOTATION="containerd.io/snapshot/stargz/toc.digest"
+UNCOMPRESSED_SIZE_ANNOTATION="io.containers.estargz.uncompressed-size"
```

Comment:
Is this a typo of "containerd" or intentional?

Reply:
This label isn't used by containerd but used by podman/CRI-O so it's intentional.